### PR TITLE
Amazon-Ecommerce-clone

### PR DESCRIPTION
**Pull request by SCR01**
Title: Add a Beginner-Friendly E-Commerce Front-End Project.
Closes: #590

Name: Sharad Chandra Reddy

Identify yourself: GSSOC'24

Describe the add-ons or changes you've made 📃

⭐Added assets and files for the project

⭐Added project folder in the readme

How Has This Been Tested? ⚙️

Tested locally

Screenshots 📷
screenshot webp

Type of change ☑️

What sort of change have you made:

[✔️] New feature (non-breaking change which adds functionality)

 
Checklist: ☑️

[✔️] My code follows the Contributing Guidelines & Code of Conduct of this project.
 
[✔️] This PR does not contain plagiarized content.
 
  [✔️] I have performed a self-review of my own code.
 
[✔️] I have commented on my code, particularly wherever it was hard to understand.
 
[✔️] My changes generate no new warnings.
 
Note to reviewers 📄

### CODE REVIEW

File: .hintrc
Comment (Avdhesh-Varshney) [inline .hintrc:1]: Delete it or keep it inside your project directory.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Carefully follow the guidelines.

**Comment by SCR01** [comment]
@Avdhesh-Varshney Please, Review this PR.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
- Naming convention of project folder as written in guidelines. `Amazon-E-Commerce-Clone`
- Add 1 ss just inside the project folder. `screenshot.webp` in webp format.
- Keep your other images inside the `assets` folder.
- And Keep your project in the `Vanilla-JS-Projects` directory ok.

**Comment by SCR01** [comment]
@Avdhesh-Varshney I did all the changes as you have mentioned, please merge it.

**Pull request closed without merging**